api/image-parser.php

```diff
@@ -38,15 +38,23 @@ function get_extracted_images() {
         /**
          * check whether URL entered by user is correct or not
          */
-        if (!isValidURL($url)) {
+        // if (!isValidURL($url)) {
 
+        //     return array(
+        //         'url_searched' => $url,
+        //         'valid_url' => false,
+        //         'success' => false
+        //     );
+
+        // }
+        if(!validate_url($url)) {
             return array(
                 'url_searched' => $url,
                 'valid_url' => false,
                 'success' => false
             );
-
-        } else {
+        }
+         else {
```

Comment:
Try to follow the existing project coding style
Both of the styles are correct, it's just that we should have a uniform style across one whole project.

Ending brackets `}` should be in same line as `else`